src/screens/BeerOfTheDay/BeerOfTheDay.jsx

```diff
@@ -7,58 +7,60 @@ import FavouriteButton from '../../shared/components/FavouriteButton/FavouriteBu
 import Button from '../../shared/components/Button/Button';
 import './styles.scss';
 
-class BeerOfTheDayComponent extends PureComponent {
-  constructor(props) {
-    super(props);
-    this.state = {
-      beer: [],
-    };
-  };
 
-  getRandomBeer = async () => {
+const createEmptyBeer = () => ({
+  name: '',
+  abv: '',
+  img: '',
+  ibu: '',
+  isPlaceholder: true,
+})
+
+function BeerOfTheDayComponent() {
+
+  const [beer, setBeer] = useState(createEmptyBeer())
```

Comment:
You should not create function to initiate state. Just pass initial object here:
```
useState({
  name: '',
  abv: '',
  img: '',
  ibu: '',
  isPlaceholder: true
})

Reply:
Done!